overlore/sqlite/vector_db.py

```diff
@@ -98,61 +112,39 @@ def get_townhalls_from_events(self, event_ids: list[int]) -> tuple[list[str], li
             - List of townhalls summary. One event_id of the list given in parameter must have been involved in the generation of the discussion.
             - Events_ids in the list given in parameter which haven't generated any summary before
         """
-        event_ids.extend([0 for i in range(0, 5 - len(event_ids))])
 
-        query = """
-            WITH GivenEventIDs(event_id) AS (VALUES (?), (?), (?), (?), (?)),
-            RecentDiscussions AS (
-                SELECT
-                    json_each.value AS event_id,
-                    T.summary,
-                    T.ts,
-                    T.rowid,
-                    ROW_NUMBER() OVER (PARTITION BY json_each.value ORDER BY T.ts DESC) as rn
-                FROM
-                    townhall T, json_each(T.events_ids)
-                WHERE
-                    json_each.value IN ((?), (?), (?), (?), (?)) AND json_each.value <> 0
```

Comment:
`json_each.value <> 0` is this check not necessary anymore with the new method?

Reply:
Not necessary anymore as we do not extend the event_ids using x amounts of '0' anymore, but adapt the query to the nb of given event_ids.